index.html

```diff
@@ -209,18 +209,21 @@
 
  <section class='container'>
   <hgroup>
-    <h1>Welcome to your static httpd application on OpenShift</h1>
+    <h1>Welcome to Riddle's static httpd web app.  I have no idea what I am going to do with this but I am learning!</h1>
   </hgroup>
   <div class="row">
     <section class='col-xs-12 col-sm-6 col-md-6'>
       <section>
         <h2>Deploying code changes</h2>
           <p>To change this application, update your code and rebuild/redeploy the image.</p>
+        <h2>These are code changes.</h2>
+          <p>Aren't they purdy?!!.</p>
       </section>
     </section>
     <section class="col-xs-12 col-sm-6 col-md-6">
       <h3>For more information and help</h3>
       <ul>
+        <li><a href="https://www.linkedin.com/in/wayne-walther-0b885711/">Wayne Walther</a></li>
```

Comment:
Adding your LinkedIn profile, really??

Reply:
probably just a PR accidentally opened against the wrong fork.  happens all the time.